config/experiment_test.go

```diff
@@ -98,15 +101,15 @@ func TestExperimentValidate(t *testing.T) {
 				AdCerts: ExperimentAdsCert{Mode: AdCertsSignerModeInprocess, InProcess: AdsCertInProcess{Origin: "http://test.com", PrivateKey: "pk", DNSCheckIntervalInSeconds: -10, DNSRenewalIntervalInSeconds: 10}},
 			},
 			expectErrors:   true,
-			expectedErrors: []error{ErrInProcessSignerInvalidDNSCheckInterval},
+			expectedErrors: []error{fmt.Errorf("invalid dns check interval for inprocess signer: -10")},
```

Comment:
`fmt.Errorf` should be used when you have a formatted error string i.e. when you're using `%s`, `%d`, etc for which the values are going to filled in run time. Here for `expectedErrors` you already have a constant string that you're expecting so you can just use `errors.New`. 

Same for L112, L123 and L124

Reply:
Agree, I missed it accidentally. Fixed.